src/components/Day/DayView.js

```diff
@@ -3,19 +3,21 @@ import PropTypes from 'prop-types';
 import { Cell } from '../Cell';
 import DayCSS from './Day.module.css';
```

Comment:
The patten when using css modules is to call the import CSS of `Styles` or just `S` (instead of `DayCSS`). That way it's predictable across all files. `Styles.container`, `Styles.cell`, etc